lib/capybara/spec/session/has_button_spec.rb

```diff
@@ -65,6 +65,18 @@
   it 'should not affect other selectors when enable_aria_role: false' do
     expect(@session).to have_button('Click me!', enable_aria_role: false)
   end
+
+  context 'with focused:', requires: [:js] do
+    it 'should be true if a field has focus when focused: true' do
+      @session.send_keys(:tab)
+
+      expect(@session).to have_button('A Button', focused: true)
+    end
+
+    it 'should be false if a field does not have focus when focused: false' do
+      expect(@session).not_to have_button('A Button', focused: false)
```

Comment:
Assuming the above test is correct and tab has to be sent to focus the button, then this test doesn't seem right to me - if the button isn't focused then it should match and it should be `expect(@session).to have_button('A button', focused: false)`